tests/sources/test_zoom.py

```diff
@@ -0,0 +1,600 @@
+    "next_page_token": None,
+    "meetings": [
+        {"id": "recording3", "type": "recording", "start_time": "2023-01-01T00:00:00Z"}
+    ],
+}
+SAMPLE_RECORDING_PAGE4 = {
+    "next_page_token": None,
+    "meetings": [
+        {"id": "recording4", "type": "recording", "start_time": "2023-12-01T00:00:00Z"}
+    ],
+}
+SAMPLE_TRASH_RECORDING = {
+    "next_page_token": None,
+    "meetings": [
+        {"id": "recording5", "type": "trash", "start_time": "2023-03-09T00:00:00Z"}
+    ],
+}
+RECORDING_EXPECTED_RESPONSE = [
+    {
+        "id": "recording1",
+        "type": "recording",
+        "start_time": "2023-03-01T00:00:00Z",
+        "_id": "recording1",
+        "_timestamp": "2023-03-01T00:00:00Z",
+    },
+    {
+        "id": "recording2",
+        "type": "recording",
+        "start_time": "2023-02-01T00:00:00Z",
+        "_id": "recording2",
+        "_timestamp": "2023-02-01T00:00:00Z",
+    },
+    {
+        "id": "recording3",
+        "type": "recording",
+        "start_time": "2023-01-01T00:00:00Z",
+        "_id": "recording3",
+        "_timestamp": "2023-01-01T00:00:00Z",
+    },
+    {
+        "id": "recording4",
+        "type": "recording",
+        "start_time": "2023-12-01T00:00:00Z",
+        "_id": "recording4",
+        "_timestamp": "2023-12-01T00:00:00Z",
+    },
+    {
+        "id": "recording5",
+        "type": "trash",
+        "start_time": "2023-03-09T00:00:00Z",
+        "_id": "recording5",
+        "_timestamp": "2023-03-09T00:00:00Z",
+    },
+]
+
+# Channel document
+SAMPLE_CHANNEL = {
+    "next_page_token": None,
+    "channels": [
+        {"id": "channel1", "type": "chat", "date_time": "2023-03-09T00:00:00Z"},
+        {"id": "channel2", "type": "channel", "date_time": "2023-02-09T00:00:00Z"},
+    ],
+}
+CHANNEL_EXPECTED_RESPONSE = [
+    {
+        "id": "channel1",
+        "type": "chat",
+        "date_time": "2023-03-09T00:00:00Z",
+        "_id": "channel1",
+        "_timestamp": "2023-03-09T00:00:00+00:00",
+    },
+    {
+        "id": "channel2",
+        "type": "channel",
+        "date_time": "2023-02-09T00:00:00Z",
+        "_id": "channel2",
+        "_timestamp": "2023-03-09T00:00:00+00:00",
+    },
+]
+
+# Chat document
+SAMPLE_CHAT = {
+    "next_page_token": None,
+    "messages": [
+        {"id": "chat1", "type": "chat", "date_time": "2023-03-09T00:00:00Z"},
+        {"id": "chat2", "type": "chat", "date_time": "2023-02-09T00:00:00Z"},
+    ],
+}
+CHAT_EXPECTED_RESPONSE = [
+    {
+        "id": "chat1",
+        "type": "chat",
+        "date_time": "2023-03-09T00:00:00Z",
+        "_id": "chat1",
+        "_timestamp": "2023-03-09T00:00:00Z",
+    },
+    {
+        "id": "chat2",
+        "type": "chat",
+        "date_time": "2023-02-09T00:00:00Z",
+        "_id": "chat2",
+        "_timestamp": "2023-02-09T00:00:00Z",
+    },
+]
+
+# File document
+SAMPLE_FILE = {
+    "next_page_token": None,
+    "messages": [
+        {
+            "file_id": "file1",
```

Comment:
in practice, are these `file_id` values unique? If we're using these for our `_id` field, we'll want to be sure that similarly named files across different meetings don't overwrite one another.